docs/toc.md

```diff
@@ -31,7 +31,8 @@
 ### [Portability Analyzer](standard/analyzers/portability-analyzer.md)
 ### [Framework Analyzer](standard/analyzers/framework-analyzer.md)
 ## [Handling and throwing exceptions](standard/exceptions/)
-## [.NET Assembly File Format](standard/assembly-format.md)
+## [Assemblies in .NET](standard/assembly/index.md)
+### [.NET Assembly File Format](standard/assembly/assembly-format.md) 
```

Comment:
one more suggestion: let's rename this article to either file-format or just format, since our SEO tools usually complain when we have repeated words in the path.